cmd/neofs-cli/modules/object/util.go

```diff
@@ -483,3 +480,72 @@ func collectObjectRelatives(ctx context.Context, cmd *cobra.Command, cli *client
 
 	return chain
 }
+
+type payloadWriter struct {
+	payload []byte
+}
+
+func (pw *payloadWriter) Write(p []byte) (n int, err error) {
+	pw.payload = append(pw.payload, p...)
+	return len(p), nil
+}
+
+func getChildrenFromV2Link(ctx context.Context, cmd *cobra.Command, cli *client.Client, key *ecdsa.PrivateKey, linkID oid.ID) ([]oid.ID, bool) {
+	var addrObj oid.Address
+	var prmGet internal.GetObjectPrm
+	pw := &payloadWriter{}
+
+	addrObj.SetObject(linkID)
+	prmGet.SetClient(cli)
+	prmGet.SetPrivateKey(*key)
+	prmGet.SetAddress(addrObj)
+	prmGet.SetRawFlag(false)
+	prmGet.SetPayloadWriter(pw)
+	Prepare(cmd, &prmGet)
+
+	_, err := internal.GetObject(ctx, prmGet)
+	if err == nil {
+		var link object.Link
+		err = link.Unmarshal(pw.payload)
+		if err != nil {
+			common.PrintVerbose(cmd, "failed to get linking object's header: %w", err)
```

Comment:
header is gotten, decode failure

but this message is also useful

Reply:
fixed